frameworks/Clojure/duct/test/hello/handler/example_test.clj

```diff
@@ -0,0 +1,4 @@
+(ns hello.handler.example-test
```

Comment:
Could you remove this test file?  I see it is basically an empty placeholder, but even if it wasn't empty, we still don't want to maintain framework-specific functional tests in this repository.

I made a similar request in another PR recently with a longer explanation:  https://github.com/TechEmpower/FrameworkBenchmarks/pull/3895#discussion_r198972038

Reply:
I removed the test file which was generated by the `lein` template.